json/tbd_warehouse6.json

```diff
@@ -0,0 +1,28 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing expansion joints for test_warehouse.osm",
+  "psis": [
+    {
+      "id": "ok",
+      "joint": 0.9
+    }
+  ],
+  "edges": [
+    {
+      "psi": "ok",
+      "type": "joint",
+      "surfaces": [
+        "Fine Storage Front Wall",
+        "Fine Storage Office Front Wall"
+      ]
+    },
+    {
+      "psi": "ok",
+      "type": "joint",
+      "surfaces": [
+        "Fine Storage Left Wall",
+        "Fine Storage Office Left Wall"
+      ]
```

Comment:
By default, TBD labels as (mild) _transitions_ edges linking parallel surfaces (e.g. between 2x butting walls sharing same 3D plane, between 2x sloping roof drainage planes). There are 2 of those in the US DOE warehouse prototype building (upper Fine Storage walls, above the office). And also by default, such transitions have a PSI-value of 0 (i.e., no derating occurs). In some cases, such intersections can represent significant thermal bridges, such as structural expansion joints or roof curbs. Users have the option of resetting the thermal bridge type e.g., from _transition_ to _joint_ (or any other type for that matter).

Reply:
I'm assuming that any explicitly listed edge types will override the automatically deduced type?  Basically, you first compute the deduced type and then go through and override with any types the user sets right?

Reply:
This should be one of the features of the UI tool.  Click on an edge, see what type is assigned.  If you don't like it, maybe the UI tool could offer to create an override for you?

Reply:
You got it. This is very similar to how OpenStudio manages surface constructions: use the building default set (if defined); use instead the story set if applicable; use instead the spacetype set if applicable, etc. down to the individual surface construction (overriding all previous default sets). TBD goes down to the individual edge.